benefits/enrollment/views.py

```diff
@@ -156,10 +156,7 @@ def success(request):
             raise Exception("EligibilityVerifier requires authentication, but OAUTH_CLIENT_NAME is None")
 
         if session.logged_in(request):
-            button = viewmodels.Button.login(
-                classes="logout",
```

Comment:
Oh, `classes` is missing from the code in the method

Reply:
Yeah - Ended up removing it b/c I am not using it for any CSS actually, and now the button is more clear b/c the method is now named "logout()".